org.eclipse.swtchart.export/src/org/eclipse/swtchart/export/menu/bitmap/JPGExportHandler.java

```diff
@@ -24,9 +25,9 @@
 
 public class JPGExportHandler extends AbstractSeriesExportHandler implements ISeriesExportConverter {
 
-	private static final String FILE_EXTENSION = "*.jpg";
-	public static final String NAME = "Image (" + FILE_EXTENSION + ")";
-	private static final String TITLE = "Save As Image";
+	private static final String FILE_EXTENSION = "*.jpg"; //$NON-NLS-1$
+	public static final String NAME = Messages.getString(Messages.IMAGE) + FILE_EXTENSION + ")"; //$NON-NLS-1$
```

Comment:
In such a case String.format or Messageformat should be used because in different languages the position of parts might differ!
You you think you can create a follow-up change?